pylint/test/unittest_checker_format.py

```diff
@@ -138,6 +138,20 @@ def testCheckKeywordParensHandlesUnnecessaryParens(self):
             with self.assertAddsMessages(msg):
                 self.checker._check_keyword_parentheses(_tokenize_str(code), offset)
 
+    def testCheckIfArgsAreUnicode(self):
+        self.checker._keywords_with_parens = set()
+        cases = [
+            (Message('superfluous-parens', line=1, args='if'),
+             'if (foo):', 0),
+            (Message('superfluous-parens', line=1, args='asssert'),
+             'assert (1 == 1)', 0)
```

Comment:
Shoudn't we test unicode-encoded code string here (`u'if (foo):'`)? From what I see those new tests will pass even without your patch.

Reply:
You are right. I lost somewhere u'' during the final edition.